migrations/sql/afterMigrate__3__permit_status_code.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO permit_status_code (
```

Comment:
Can you make this as INSERT INTO ..... ON Conflict DO NOTHING kind of syntax.
Coz the afterMigrate scrips run after each deploy. So the second time they run, it'll throw a Primary Key unique constraint error.
More deets here on implementation:
https://stackoverflow.com/questions/17267417/how-to-upsert-merge-insert-on-duplicate-update-in-postgresql